beacon_node/execution_layer/src/lib.rs

```diff
@@ -222,6 +222,11 @@ struct Inner<E: EthSpec> {
     builder_profit_threshold: Uint256,
     log: Logger,
     always_prefer_builder_payload: bool,
+    /// Track whether the last `newPayload` call errored.
+    ///
+    /// This is used *only* in the informational sync status endpoint, so that a VC using this
+    /// node can prefer another node with a healthier EL.
+    last_new_payload_errored: RwLock<bool>,
```

Comment:
I had originally expected that we'd track the last fcU error rather than the last newPayload error.

My reasoning was that it's the *last* call we'd do in the block import process so it has the most up-to-date information about EE state. However I see now that if we fail a newPayload then we won't end up calling fcU and we'd end up kinda stuck.

If we're choosing just one, then I now prefer newPayload. No changes suggested, just sharing my reasoning.